integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/context/TestContext.java

```diff
@@ -9,58 +14,56 @@
 
     private String managerToken;
 
-    private String randomBridgeName;
-    private String bridgeId;
-    private String processorId;
-    private String endPoint;
+    private Map<String, BridgeContext> bridges = new HashMap<>();
+    private Map<String, BridgeContext> removedBridges = new HashMap<>();
 
     private Scenario scenario;
 
     public TestContext() {
     }
 
     public String getManagerToken() {
-        return managerToken;
+        return this.managerToken;
     }
 
     public void setManagerToken(String managerToken) {
         this.managerToken = managerToken;
     }
 
-    public String getRandomBridgeName() {
-        return randomBridgeName;
-    }
-
-    public void setRandomBridgeName(String randomBridgeName) {
-        this.randomBridgeName = randomBridgeName;
-    }
-
-    public String getBridgeId() {
-        return bridgeId;
-    }
-
-    public void setBridgeId(String bridgeId) {
-        this.bridgeId = bridgeId;
+    public BridgeContext newBridge(String testBridgeName, String systemBridgeName) {
```

Comment:
May be good to add a javadoc explaining what is `testBridgeName` and `systemBridgeName`

Reply:
added